### DISCUSSION TITLE
umi3/4 有没有内置对 keep-alive 的计划，或者时间点？

### DISCUSSION THREAD

**Discussion by hoperyy** [umijs/umi#11224] [Q&A] [answered]
umi3/4 有没有内置对 keep-alive 的计划，或者时间点？
第三方实现的成本蛮高的。

**Comment by hoperyy** [comment]
类似：https://micro-zoe.com/docs/1.x/#/zh-cn/keep-alive

```html
<micro-app name='xx' url='xx' keep-alive></micro-app>
```

标签上加一个 keep-alive 属性即可

**Reply by fz6m** [reply, marked as answer]
目前没有计划内置，但社区有一定的解法：https://github.com/alitajs/alita/blob/master/packages/plugins/src/keepalive.ts

参考文章：https://juejin.cn/post/7109492504424087566
